src/components/Admin/ManageSensors/SensorTable/ErrorTags.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  Popover,
+  PopoverTrigger,
+  Button,
+  Portal,
+  PopoverArrow,
+  PopoverContent,
+  PopoverHeader,
+  PopoverCloseButton,
+  PopoverBody,
+  Text,
+  Heading,
+  Flex,
+} from '@chakra-ui/react';
+import {WarningTwoIcon} from '@chakra-ui/icons';
+
+/**
+ * Interface for props for ErrorTag
+ * - `name` - name of the error
+ * - `explanation` - explanation of the error
+ */
+interface ErrorTagProps {
+  name: string;
+  explanation: string;
+}
+
+/**
+ * Tag for an error for the sensor table. When clicked on, a popover appears
+ * with an explanation of the error.
+ * @param name - name of the error
+ * @param explanation - explanation of the error
+ * @returns tag for an error to be displayed in the sensor table. When clicked, the tag will show a popover with an explanation of the error.
+ */
+const ErrorTag: ({name, explanation}: ErrorTagProps) => JSX.Element = ({
+  name,
+  explanation,
+}: ErrorTagProps) => {
+  return (
+    <Popover>
+      <PopoverTrigger>
+        <Button colorScheme="red" size="xs">
```

Comment:
nit: consider not red? I know red is  the "error color", so this is totally opinion (and maybe a bad one). I just felt like it might be nice if the error buttons were differentiated from the delete-relates buttons we already have on the page.

Reply:
What color do you think would work well?